.npmrc

```diff
@@ -1,6 +1,6 @@
 git-checks = true
 hoist-pattern[] = jest-runner
-link-workspace-packages = true
+link-workspace-packages = false
```

Comment:
Why close this config here?

Reply:
I believe there was an issue with a utility library

it might not be needed after a rebase but we are blocked until Jest has no full support for esmodules